redis_aamks/worker/config.py

```diff
@@ -1,7 +1,7 @@
 import os
 redis_host = os.environ['AAMKS_SERVER']
 redis_password = os.environ['AAMKS_REDIS_PASS']
-main_path = "/".join(os.environ['AAMKS_PROJECT'].split('/')[:-3])
+# main_path = "/".join(os.environ['AAMKS_PROJECT'].split('/')[:-3])
```

Comment:
main_path is used to logger, you should remove #

Reply:
Ok, I restore logger. Logger doesnt work on our test environment (83,84 machines) - there is a problem with environment variables. There will be an exception when reading environment variables.